<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>d_editor</class>
 <widget class="QDialog" name="d_editor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>7</number>
     </property>
     <widget class="QWidget" name="tab_editor">
      <attribute name="title">
       <string>Editor</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QPlainTextEdit" name="mem_editor"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_session">
      <attribute name="title">
       <string>Sessions</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="bt_sessions_exec">
            <property name="text">
             <string>view</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbo_sessions">
            <item>
             <property name="text">
              <string>ACTIVE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>INACTIVE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>%</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="tree_sessions">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="mem_sessions"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_explain">
      <attribute name="title">
       <string>Explain</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QPushButton" name="bt_explain">
          <property name="geometry">
           <rect>
            <x>1</x>
            <y>2</y>
            <width>80</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>Run</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="mem_explain"/>
       </item>
       <item>
        <widget class="QTreeWidget" name="tree_explain">
         <property name="columnCount">
          <number>7</number>
         </property>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">2</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">3</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">4</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">5</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">6</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string notr="true">7</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_csv_updater">
      <attribute name="title">
       <string>CSV Updater</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Options</string>
         </property>
         <widget class="QPushButton" name="bt_csv_of">
          <property name="geometry">
           <rect>
            <x>596</x>
            <y>64</y>
            <width>32</width>
            <height>26</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
         <widget class="QPushButton" name="bt_csv_populate">
          <property name="geometry">
           <rect>
            <x>12</x>
            <y>32</y>
            <width>100</width>
            <height>26</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Populate</string>
          </property>
         </widget>
         <widget class="QCheckBox" name="chk_csv">
          <property name="geometry">
           <rect>
            <x>118</x>
            <y>33</y>
            <width>510</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>First Line = Titles</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="edt_csv_file">
          <property name="geometry">
           <rect>
            <x>12</x>
            <y>64</y>
            <width>578</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QTableWidget" name="grid_csv">
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>300</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="columnCount">
              <number>1</number>
             </property>
             <column/>
            </widget>
            <widget class="QPlainTextEdit" name="mem_csv">
             <property name="font">
              <font>
               <family>Monospace</family>
              </font>
             </property>
             <property name="lineWrapMode">
              <enum>QPlainTextEdit::NoWrap</enum>
             </property>
             <property name="backgroundVisible">
              <bool>false</bool>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_find_objects">
      <attribute name="title">
       <string>Find Objects</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>9</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>9</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Filter</string>
         </property>
         <widget class="QCheckBox" name="chk_obj_table">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>30</y>
            <width>85</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>TABLE</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="chk_obj_view">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>50</y>
            <width>85</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>VIEW</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="chk_obj_package">
          <property name="geometry">
           <rect>
            <x>390</x>
            <y>30</y>
            <width>85</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>PACKAGE</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="chk_obj_trigger">
          <property name="geometry">
           <rect>
            <x>290</x>
            <y>70</y>
            <width>85</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>TRIGGER</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="chk_obj_procedure">
          <property name="geometry">
           <rect>
            <x>390</x>
            <y>70</y>
            <width>131</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>PROCEDURE</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="chk_obj_funcion">
          <property name="geometry">
           <rect>
            <x>390</x>
            <y>50</y>
            <width>85</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>FUNCTION</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLineEdit" name="edt_objetos">
          <property name="geometry">
           <rect>
            <x>13</x>
            <y>53</y>
            <width>241</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>13</x>
            <y>33</y>
            <width>91</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>Object Name</string>
          </property>
         </widget>
         <widget class="QPushButton" name="bt_obj_viewcode">
          <property name="geometry">
           <rect>
            <x>510</x>
            <y>40</y>
            <width>161</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>View Code (DDL)</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="grid_objetos">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_config">
      <attribute name="title">
       <string>Configs</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>200</height>
          </size>
         </property>
         <property name="title">
          <string> Configs</string>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>30</y>
            <width>121</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>Oracle Home</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="edt_cfg_oracle_dir">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>50</y>
            <width>231</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QCheckBox" name="chk_cfg_dark_mode">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>50</y>
            <width>151</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Themed Dark Mode</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>80</y>
            <width>161</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>Templates Directory</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="edt_cfg_template_dir">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>100</y>
            <width>231</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
         <widget class="QCheckBox" name="chk_cfg_run_user">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>70</y>
            <width>191</width>
            <height>24</height>
           </rect>
          </property>
          <property name="text">
           <string>Default &quot;Run User&quot; </string>
          </property>
         </widget>
         <widget class="QPushButton" name="bt_cfg_salvar">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>130</y>
            <width>151</width>
            <height>41</height>
           </rect>
          </property>
          <property name="text">
           <string>Save Configs</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>130</y>
            <width>161</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>Outputs Directory</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="edt_cfg_output_dir">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>150</y>
            <width>231</width>
            <height>26</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="title">
          <string>TNS</string>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTabWidget" name="tabWidget_2">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <widget class="QWidget" name="tab_cfg_services_names">
             <attribute name="title">
              <string>Services Names/SIDs</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QPlainTextEdit" name="mem_cfg_tns">
                <property name="font">
                 <font>
                  <family>Monospace</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="lineWrapMode">
                 <enum>QPlainTextEdit::NoWrap</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_cfg_users_saved">
             <attribute name="title">
              <string>Saved Users</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QPlainTextEdit" name="mem_cfg_tns_saved">
                <property name="font">
                 <font>
                  <family>Monospace</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="lineWrapMode">
                 <enum>QPlainTextEdit::NoWrap</enum>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_recall">
      <attribute name="title">
       <string>Recall</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Filter Query</string>
         </property>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>30</y>
            <width>81</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>type text:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="edt_recall">
          <property name="geometry">
           <rect>
            <x>160</x>
            <y>50</y>
            <width>301</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>81</width>
            <height>18</height>
           </rect>
          </property>
          <property name="text">
           <string>database:</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="edt_recall_database">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>131</width>
            <height>26</height>
           </rect>
          </property>
          <property name="text">
           <string>%</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="grid_recall">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Date</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Database</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>SQL</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="mem_recall"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_importer">
      <attribute name="title">
       <string>Importer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>100</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Filter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="mem_sessions_2"/>
       </item>
       <item>
        <widget class="QTableWidget" name="grid_objetos_2">
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
